instrumentation/grpc-1.40.0/src/main/java/com/nr/agent/instrumentation/grpc/GrpcUtil.java

```diff
@@ -34,14 +34,14 @@ public static void finalizeTransaction(Token token, Status status, Metadata meta
     }
 
     /**
-     * Set the response.status attribute and error cause (if applicable) on the transaction
+     * Set the http.statusCode attribute and error cause (if applicable) on the transaction
      * when the ServerStream is closed or cancelled.
      *
      * @param status The {@link Status} of the completed/cancelled operation
      */
     public static void setServerStreamResponseStatus(Status status) {
         if (status != null) {
-            NewRelic.addCustomParameter("response.status", status.getCode().value());
+            NewRelic.addCustomParameter("http.statusCode", status.getCode().value());
```

Comment:
Nice catch. This instrumentation module was around the same time the attribute name change happened, but the author did not notice that.